src/main/flight/pid.c

```diff
@@ -93,6 +93,21 @@ void pidResetErrorGyro(void)
 
 const angle_index_t rcAliasToAngleIndexMap[] = { AI_ROLL, AI_PITCH };
 
+
+// Low Pass filter for PID controllers
+void pt1_element(int axis, float *delta, uint8_t f_cut, int type) {
+
+	float dT = (float)cycleTime * 0.000001f;
+
+    float    RC;
+    static float    lastDelta[][3] = {{ 0.0f, 0.0f, 0.0f }};
+
```

Comment:
You can not do this
```C
(float*) &DTerm
```
when DTerm is in reality a int32_t type. This way you tell the compiler that the thing in memory is a float, while it is not a float. Your code will then use this apple as a pear (so to speak).